### PR TITLE
PPTP-1401 | Exported components used for human meds

### PR DESCRIPTION
**Pull request by liampan**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: migrate.sh [hunk 1]
```diff
@@ -9,7 +9,9 @@ do
     echo "Applying migration $file"
     chmod u+x $file
     /bin/bash $file
-    mv $file ./applied_migrations
+    echo ""
+    echo "deleting migration file $file"
+    rm $file
```
Comment (liampan) [inline migrate.sh:14]: did a thing, hope its ok 👀

Comment (JoeSimmonds) [reply migrate.sh:14]: I think the latest version of scaffolds does that now,  keeping the migrations is pretty pointless. You can't undo them

**Review by C-Thomp — APPROVED** [review]
all good

**Comment by liampan** [comment]
Closing as #235 contains all of this pr and is being merged now.

**Pull request closed without merging**